BackgroundChanger.js

```diff
@@ -0,0 +1,31 @@
+const TIME = 10;
```

Comment:
this constant leaves too much room for interpretation, can you choose a better constant name (something that makes a person confident when changing the value)